apiroutes.go

```diff
@@ -923,6 +925,23 @@ func (c *appContext) getBlockRangeSteppedSummary(w http.ResponseWriter, r *http.
 	fmt.Fprintf(w, "]")
 }
 
+func (c *appContext) getTicketPool(w http.ResponseWriter, r *http.Request) {
+	// getBlockHeightCtx falls back to try hash if height fails
+	idx := c.getBlockHeightCtx(r)
+	if idx < 0 {
+		http.Error(w, http.StatusText(422), 422)
+		return
+	}
+
+	tp, err := c.BlockData.GetPool(int(idx))
```

Comment:
We should use the correct type instead of casting.

Reply:
agreed, `GetPool` should take `int64` since it does a conversion too lol